projects/ziti-console-lib/src/lib/pages/services/services-page.component.ts

```diff
@@ -0,0 +1,133 @@
+import {Component, Inject, OnInit, OnDestroy} from '@angular/core';
```

Comment:
i won't make this comment again- missing copyright. please check all net-new files for that header.

Reply:
Will do